templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
```

Comment:
Looks like you might be right:

> database.yml present
> DATABASE_URL present
> => Merged into `url` sub key. If both specify `url` sub key, the `database.yml` `url` 
>     sub key "wins". If other paramaters `adapter` or `database` are specified in YAML, 
>     they are discarded as the `url` sub key "wins".

https://github.com/rails/rails/pull/13463#issuecomment-31480799 linked from https://github.com/rails/rails/pull/13578

Which would mean that we can't use defaults in production if it specifies database names, etc. but can set things like pool to be merged in.

These are the keys we already have which I think we can set safely, without overriding DATABASE_URL:

```yaml
  encoding: utf8
  min_messages: warning
  pool: 2
  timeout: 5000
```